src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -787,7 +805,7 @@ private SharedPreferences getSharedPreferences() {
         private final String mToken;
         private final Lock mStartLock;
 
-        private final List<Pair<String,JSONObject>> mEditorChanges;
+        private final Map<String, Pair<String,JSONObject>> mEditorChanges;
```

Comment:
hmmm.. I'm a bit confused.. mEditorChanges is declared as a Map but initialized as a HashMap. is this intentional?